user_sync/config.py

```diff
@@ -84,6 +87,24 @@ def set_options(self, caller_options):
     def get_logging_config(self):
         return self.main_config.get_dict_config('logging', True)
 
+    def validate_limits_config(self):
+        ''' Ensure that we have limits set for max deletions per run and max missing users
+        '''
+        limits_config = self.main_config.get_dict_config('limits', True)
+        if limits_config == None:
+            raise user_sync.error.AssertionException("Limits configuration must be specified.")
+
+        max_deletions_per_run = limits_config.get_int('max_deletions_per_run',True)
```

Comment:
Normally, I would not allow None to be allowed (i.e. limits_config.get_int('max_deletions_per_run')) so that an exception would be raised.   This is preferable since we could modify the internal message with the location via get_full_scope() in the future.